databases/week2/PhoneStore Database.sql

```diff
@@ -0,0 +1,35 @@
+CREATE DATABASE phoneStore;
+
+USE phoneStore;
+
+CREATE TABLE phones (
+    `phone_id` INT PRIMARY KEY,
```

Comment:
What about adding AUTO_INCREMENT here? So you won't need to care of assigning id's every time, it will also minimize the chance of making a mistake with the id. 🤓